docs/runtimes/images.md

```diff
@@ -20,12 +21,12 @@ Every release of MLRun includes several images for different usages. All images
 
 The images are:
 
-`mlrun/mlrun`: An MLRun image includes preinstalled OpenMPI. Useful as a base image for simple jobs.
-`mlrun/mlrun-gpu`: The same as `mlrun/mlrun` but for GPUs, including Open MPI. 
-`mlrun/ml-base`: The image for file acquisition, compression, dask jobs, simple training jobs and other utilities. Like `mlrun/mlrun` with the addition of Miniconda and other python packages.
-`mlrun/jupyter`: An image with Jupyter giving a playground to use MLRun in the open source. Built on top of jupyter/scipy-notebook, with the addition of MLRun and several demos and examples.
-`mlrun/mlrun-api`: The image used for running the MLRun API.
-`mlrun/mlrun-ui`: The image used for running the MLRun UI.
+- `mlrun/mlrun`: An MLRun image includes preinstalled OpenMPI. Useful as a base image for simple jobs.
+- `mlrun/mlrun-gpu`: The same as `mlrun/mlrun` but for GPUs, including Open MPI. 
+- `mlrun/ml-base`: The image for file acquisition, compression, dask jobs, simple training jobs and other utilities. Like `mlrun/mlrun` with the addition of Miniconda and other python packages.
+- `mlrun/jupyter`: An image with Jupyter giving a playground to use MLRun in the open source. Built on top of jupyter/scipy-notebook, with the addition of MLRun and several demos and examples.
+- `mlrun/mlrun-api`: The image used for running the MLRun API.
+- `mlrun/mlrun-ui`: The image used for running the MLRun UI.
```

Comment:
These 2 are not related to runtimes images. Don't think they should be here.

Reply:
deleted, and changed the heading name to **MLRun runtime images**